swap_meet/vendor.py

```diff
@@ -1,2 +1,92 @@
 class Vendor:
-    pass
\ No newline at end of file
+    # Wave 1 
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.item = item
+        self.inventory.append(self.item)
+        return self.item
+
+    def remove(self, item):
+        self.item = item
+        if self.item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(self.item)
+            return self.item
+
+    # Wave 2 
+    def get_by_category(self, category):
+        items_filtered_by_category = []   
+
+        for item in self.inventory:
+            if item.category == category:
+                items_filtered_by_category.append(item)
+
+        return items_filtered_by_category
```

Comment:
Awesome job! As an alternative, can you consider what this would look like with a list comprehension? Which do you prefer?